mcs/tools/mdoc/Mono.Documentation/monodocer.cs

```diff
@@ -274,51 +324,136 @@ public override void Run (IEnumerable<string> args)
 				"Do not delete members that don't exist in the assembly, but rather mark them as preserved.",
 				v => PreserveTag = "true" },
 		};
-		var assemblies = Parse (p, args, "update", 
+		var assemblyNames = Parse (p, args, "update", 
 				"[OPTIONS]+ ASSEMBLIES",
 				"Create or update documentation from ASSEMBLIES.");
-		if (assemblies == null)
+		if (assemblyNames == null)
 			return;
-		if (assemblies.Count == 0)
+		if (assemblyNames.Count == 0)
 			Error ("No assemblies specified.");
 
-		foreach (var dir in assemblies
-				.Where (a => a.Contains (Path.DirectorySeparatorChar))
-				.Select (a => Path.GetDirectoryName (a)))
-			assemblyResolver.AddSearchDirectory (dir);
+		ProcessAssemblyOwners (ref assemblyNames);
+
+		// check to see if there are two dropns entries
```

Comment:
Well ... the purpose of the code is, specifically, to support a combined ios and mac  set of documentation, so the true purpose was always for two sets of assemblies from which you're dropping a namespace. I added support for one to make writing the unit tests easier and is how I developed the bulk of the functionality initially (tdd) ... but any more than that delves into territory that I didn't test, and didn't need to test, so if there are more than two it just assumes you're not dropping a namespace.